contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -384,6 +374,7 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
         );
 
         // Get queue length for future comparison/
+        iOVM_ChainStorageContainer queueRef = queue();
```

Comment:
Consider these changes to cut down on calls to the address manager: https://github.com/ethereum-optimism/contracts-v2/pull/205/files#diff-c3f43bf36b5d31fd82bda152c3e2fd95880d2ded92a46c7363b8463d6d2acd0fR161-R164

`getQueueLength` has a `resolve()` in it, i think we should never call it internally